modules/templates/helper.go

```diff
@@ -620,6 +622,17 @@ func Str2html(raw string) template.HTML {
 	return template.HTML(markup.Sanitize(raw))
 }
 
+// Markdown2html render Markdown text to HTML (non-sanitize)
+func Markdown2html(raw string) template.HTML {
+	var err error
+	var renderedContent string
+	if renderedContent, err = markdown.RenderString(&markup.RenderContext{}, raw); err != nil {
+		log.Warn("Markdown2html: Invalid markdown? %v", err)
+		return template.HTML(html.EscapeString(raw))
+	}
+	return template.HTML(renderedContent)
```

Comment:
```suggestion
	return template.HTML(markup.Sanitize(renderedContent))
```

After reading other markup related code, I think we need the `Sanitize` here (and the comment would be update to `render Markdown text to sanitized HTML`)

Otherwise LGTM ~~